etc/update-spec-tests.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+# This script is used to fetch the latest JSON tests for the CRUD spec. It puts the tests in the
```

Comment:
nit: these details aren't correct, it's not specific to the CRUD tests, and the $reproroot variable is not used.

Reply:
Updated! Thanks 😃